schema/deploy/functions/update-modified-timestamp.sql

```diff
@@ -0,0 +1,18 @@
+-- Deploy seattleflu/schema:functions/update-modified-timestamp to pg
+
+begin;
+
+create or replace function warehouse.update_modified_timestamp() returns trigger as $$
+
+    begin
+        NEW.modified = now();
+        return new;
+    end;
+$$ language plpgsql
```

Comment:
This function should be marked as `stable` instead of the default of `volatile` since it only uses stable functions and doesn't have side-effects. More info in the Pg docs: https://www.postgresql.org/docs/10/xfunc-volatility.html

Reply:
Neat to read about the optimization advantages of marking functions as `stable` or `immutable`.